packages/tocco-ui/src/EditableValue/typeEditors/Document.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Document = props => {
+  return (
+    <div className="form-control-static">
+      <a href={props.value.binaryLink} download>
+        {props.value.fileName}
+      </a>
+    </div>
+  )
+}
+
+Document.propTypes = {
+  value: React.PropTypes.object
```

Comment:
Right now this component is not exposed and can only be used inside <EditableValue>. a null-check is handled there. So value can not be null. 
Setting it as required is a good idea. Should be done on any types implementation though..